keras2onnx/ke2onnx/upsample.py

```diff
@@ -29,6 +29,12 @@ def convert_keras_upsample(scope, operator, container, n_dims):
     else:
         raise ValueError('Unsupported dimension %s when converting Keras Upsampling layer' % n_dims)
 
+    mode = 'nearest'
+    if hasattr(op, 'interpolation'):
+        mode = op.interpolation
+        if mode.endswith('linear'):
+            mode = 'linear'
```

Comment:
mode = 'linear' if mode.endswith('linear') else op.interpolation

Reply:
resolve this?

Reply:
please rewrite this piece of code concisely like: mode = 'linear' if mode.endswith('linear') else op.interpolation

Reply:
Done.